.github/workflows/release.yaml

```diff
@@ -0,0 +1,78 @@
+name: build
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+  push:
```

Comment:
I might be wrong, but we probably only want this on tag releases right?

Reply:
i think in this file, there's 2... a release job down below looks for the tags, but the test suite always look for the PR to be opened and synchronized.